src/common/resources/ledger.js

```diff
@@ -8,7 +8,7 @@ export const ledgersResource = {
   params: {
     query: 'cql.allRecords=1 sortby name',
   },
-  records: 'ledgers',
```

Comment:
hmmm. this might affect other places that use this resource. Please keep that in mind

Reply:
yep, no need to change common resource